db/migrations/0002_cinemahall_movie_moviesession.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.0.2 on 2024-10-29 14:07
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CinemaHall',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('rows', models.IntegerField()),
+                ('seats_in_row', models.IntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Movie',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=255)),
+                ('description', models.TextField()),
```

Comment:
The `ManyToManyField` for `actors` references `Actor`, but it should include the app label if `Actor` is not in the same app. Ensure the correct app label is used, like `to='app_label.Actor'`.